scene/3d/physics_body_3d.cpp

```diff
@@ -1584,6 +1584,7 @@ void CharacterBody3D::_set_collision_direction(const PhysicsServer3D::MotionResu
 	Vector3 prev_wall_normal = wall_normal;
 	int wall_collision_count = 0;
 	Vector3 combined_wall_normal;
+	Vector3 tmp_wall_col = Vector3(); // Avoid duplicate on average calculation.
```

Comment:
```suggestion
	Vector3 tmp_wall_col; // Avoid duplicate on average calculation.
```
`Vector3` is already default initialized to 0.

Reply:
Always the same mistake, I promise one day I'll remember it!!